fe/fe-core/src/main/java/com/starrocks/http/BaseAction.java

```diff
@@ -321,8 +321,13 @@ public static UserIdentity checkPassword(ActionAuthorizationInfo authInfo)
                 globalStateMgr.getAuthenticationMgr().checkPlainPassword(
                         authInfo.fullUserName, authInfo.remoteIp, authInfo.password);
         if (currentUser == null) {
+            String userAndHost = authInfo.fullUserName.concat("@").concat(authInfo.remoteIp);
+            if (globalStateMgr.getAuthenticationMgr().isUserLocked(userAndHost)) {
+                throw new AccessDeniedException("Access denied for " + userAndHost + ". Locked for " +
+                    globalStateMgr.getAuthenticationMgr().getRemainingLockedTime(userAndHost) + " seconds.");
```

Comment:
not sure it is good or not to show end user the seconds remaining for next attempt.